week_1/css_advanced_and_best_practices/movie.html

```diff
@@ -3,66 +3,66 @@
 <head>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="https://cdn.tailwindcss.com"></script>
```

Comment:
**Suggestion:** The script tag for Tailwind CSS is not correct. It should be replaced with the correct CDN link for Tailwind CSS.
```suggestion
  <link href="https://cdn.tailwindcss.com/2.2.19/tailwind.min.css" rel="stylesheet">
```